### ISSUE TITLE
Warning every 6 seconds

### ISSUE THREAD

**Issue by txwindsurfer** [mvdwetering/ytmdesktop_remote#1]
If the desktop application is not running, this nice remote control give a warning every 6 seconds and fills the log queue.  Instead, it would be preferable to mark the device/entity as unavailable and issue 1 warning.  A reload of the integration when the desktop application is started is all that is needed when the desktop application is running again

```
Logger: homeassistant.components.media_player
Source: helpers/entity_platform.py:855
Integration: Media player (documentation, issues)
First occurred: 2:17:26 PM (21 occurrences)
Last logged: 2:19:26 PM

Updating ytmdesktop_remote media_player took longer than the scheduled update interval 0:00:06
```

**Comment by mvdwetering** [comment]
Yeah I know :( 

It has been on the list of things to fix eventually for quite some time now. I will see if I can fix it somewhere in the next few weeks.

btw, didn't realize more people were using this integration as I never advertised it anywhere.

**Comment by mvdwetering** [comment]
That was actually easier than anticipated. I just released 0.5.0 which solves the log spam.

**Comment by txwindsurfer** [comment]
Hi Michael,

Thanks so much for the changes. I may be the only one using and wanted to be sure to send my appreciation.

Best regards
Reagan

________________________________
